src/dataflow/utils.h

```diff
@@ -32,11 +32,7 @@
 namespace wasm::DataFlow {
 
 inline std::ostream& dump(Node* node, std::ostream& o, size_t indent = 0) {
-  auto doIndent = [&]() {
-    for (size_t i = 0; i < indent; i++) {
-      o << ' ';
-    }
-  };
+  auto doIndent = [&]() { o << std::string(indent, ' '); };
```

Comment:
I like this change, but is it connected to any other changes in this PR? If not, can we separate it into a separate PR?

Reply:
Sure! Will separate to another PR